src/lib/storage/index/partial_hash/partial_hash_index_impl.cpp

```diff
@@ -134,7 +31,10 @@ size_t PartialHashIndexImpl<DataType>::insert_entries(
       const auto row_id = RowID{chunk.first, position.chunk_offset()};
       // If value is NULL, add to NULL vector, otherwise add into value map.
       if (position.is_null()) {
-        _null_values.emplace_back(row_id);
+        if (!_null_values.contains(DataType{})) {
+          _null_values[DataType{}] = std::vector<RowID>();
```

Comment:
It can not happen we want to index DataType{}, can it?

Reply:
If a tuple that we want to index has the value `DataType{}`, it would not be stored in `_null_values` since it is not a null value. Thus, the RowID to that tuple would be stored in the map for non-null values.